src/views/Login.jsx

```diff
@@ -11,6 +13,14 @@ export class Login extends React.Component {
     }
   }
 
+  componentWillReceiveProps (nextProps, nextContext) {
+    // if token and user exists, then we're already logged in so redirect
+    // TODO: validate token
+    if (nextProps.user && nextProps.token) {
+      // TODO redirect
```

Comment:
having trouble actually figuring out how to redirect. online docs say to use the `router` from `context`, but it looks like our `router` is an object instead of a function?